DQMOffline/Configuration/python/DQMOffline_SecondStep_cff.py

```diff
@@ -98,8 +98,8 @@
                                          DQMMessageLoggerClientSeq )
 
 
-HLTMonitoringClient = cms.Sequence(trackingMonitorClientHLT * trackingForDisplacedJetMonitorClientHLT)
-HLTMonitoringClientPA= cms.Sequence(trackingMonitorClientHLT * PAtrackingMonitorClientHLT)
+HLTMonitoringClient = cms.Sequence(trackingMonitorClientHLT * trackEfficiencyMonitoringClientHLT * trackingForDisplacedJetMonitorClientHLT)
+HLTMonitoringClientPA= cms.Sequence(trackingMonitorClientHLT * trackEfficiencyMonitoringClientHLT * PAtrackingMonitorClientHLT)
```

Comment:
I think one would also have to add `hltToOfflineTrackValidatorSequence` to `OfflineHLTMonitoringPA` in order to produce the histograms used by `trackEfficiencyMonitoringClientHLT` in `HLTMonitoringClientPA`.

In any case, since this PR is now considered urgent (https://github.com/cms-sw/cmssw/pull/38959#issuecomment-1206178856), feel free to postpone HIon-related updates.

Reply:
Yes. In fact, one would need a dedicated sequence, similar to ```hltToOfflineTrackValidatorSequence```  but with the collections that need to be monitored in the Heavy-Ions scenario. I would indeed suggest to leave these changes for a separate PR, either by or with further input from Heavy-Ions experts